api/src/main/java/org/apache/cloudstack/api/command/admin/host/AddHostCmd.java

```diff
@@ -54,7 +54,7 @@
     @Parameter(name = ApiConstants.CLUSTER_NAME, type = CommandType.STRING, description = "the cluster name for the host")
     private String clusterName;
 
-    @Parameter(name = ApiConstants.PASSWORD, type = CommandType.STRING, required = true, description = "the password for the host")
+    @Parameter(name = ApiConstants.PASSWORD, type = CommandType.STRING, description = "the password for the host")
```

Comment:
@Pearl1594 
should we update the description if they are required by kvm/xenserver or other hypervisors ?